services-camel-app/src/main/java/edu/unc/lib/boxc/services/camel/images/Jp2Processor.java

```diff
@@ -0,0 +1,26 @@
+package edu.unc.lib.boxc.services.camel.images;
+
+import JP2ImageConverter.CLIMain;
+import edu.unc.lib.boxc.services.camel.util.CdrFcrepoHeaders;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Jp2Processor implements Processor {
+    private static final Logger log = LoggerFactory.getLogger(Jp2Processor.class);
+
+    @Override
+    public void process(Exchange exchange) throws Exception {
+        Message in = exchange.getIn();
+        String imagePath = (String) in.getHeader(CdrFcrepoHeaders.CdrImagePath);
+        String tempPath = (String) in.getHeader(CdrFcrepoHeaders.CdrTempPath);
+        String mimetype = (String) in.getHeader(CdrFcrepoHeaders.CdrBinaryMimeType);
+
+        String[] command = new String[]{"jp24u", "kdu_compress", "-f", "\"" + imagePath + "\"",
```

Comment:
My understanding is that when we are directly invoking the picocli interface like this with an array of params, it doens't need to be quoted like it would if being executed via a ProcessBuilder, since its not actually calling out to a shell. So I think we can remove all the `"\""` bits and just pass in the variables directly, since picocli knows they are separate arguments since they are separate array elements. Its worth a try at least, and would avoid us having to add in quote escaping in case paths or mimetypes had quotes in them.